<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindowClass</class>
 <widget class="QMainWindow" name="HomeWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1904</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {

	background-color: rgb(149, 99, 74);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:white;</string>
    </property>
    <property name="text">
     <string>Employee Management System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>611</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="login_btn">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="create_btn">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
       </property>
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="main_widget" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1904</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <addaction name="menuTheme"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HomeWindow.qrc"/>
 </resources>
 <connections/>
</ui>
